tajo-catalog/tajo-catalog-common/src/main/java/org/apache/tajo/catalog/statistics/StatisticsUtil.java

```diff
@@ -83,6 +83,14 @@ public static void aggregateTableStat(TableStats result, TableStats stats) {
       }
     }
 
+    // If there is partitions
```

Comment:
TableStats may be not a good place to contain added partitions because partition is not a part of statistic information. We need to improve it here or later. How do you think of it?